dts/x86/Makefile

```diff
@@ -1,6 +1,7 @@
 ifeq ($(CONFIG_HAS_DTS),y)
-dtb-$(CONFIG_SOC_QUARK_SE_C1000)	=	arduino_101.dts_compiled
-dtb-$(CONFIG_BOARD_QUARK_D2000_CRB)	=	quark_d2000_crb.dts_compiled
-dtb-$(CONFIG_BOARD_TINYTILE)		=	tinytile.dts_compiled
+dtb-$(CONFIG_SOC_QUARK_SE_C1000)		=	arduino_101.dts_compiled
+dtb-$(CONFIG_BOARD_QUARK_D2000_CRB)		=	quark_d2000_crb.dts_compiled
+dtb-$(CONFIG_BOARD_TINYTILE)			=	tinytile.dts_compiled
+dtb-$(CONFIG_BOARD_QUARK_SE_C1000_DEVBOARD)	=	quark_se_c1000_devboard.dts_compiled
```

Comment:
what is the reason for having all use CONFIG_BOARD excluding arduino 101 which uses CONFIG_SOC?